python/cuspatial/cuspatial/tests/binpreds/binpred_test_dispatch.py

```diff
@@ -326,6 +326,78 @@ def predicate(request):
         LineString([(0.5, 1.25), (0.5, -0.25)]),
         point_polygon,
     ),
+    "linestring-polygon-cross-concave-edge": (
+        """
+    -----
+    x\\-/x
+    | x |
+    |   |
+    -----
+    """,
```

Comment:
I can't tell what this (or most of the other) ascii art is showing.

Reply:
It certainly isn't as informative as the graphs produced by `shapely` but I can't insert those here. Really they're just intended as a visual guide to make it easier to parse the specific geometry definitions that come immediately afterward, instead of having to do all mental or all on-paper arithmetic to reproduce what's being tested:
```
    "linestring-polygon-cross-concave-edge": (
        """
    -----
    x\\-/x
    | x |
    |   |
    -----
    """,
        LineString([(0.5, 0.0), (0.5, 1.0)]),
        Polygon([(0, 0), (0, 1), (0.3, 0.4), (1, 1), (1, 0)]),
    ),
```
This one in particular is just plain wrong though so I've fixed that:
```
    x  x  x
    |\ | /|
    | xx- |
    |  |  |
    ---x---
```
Which corresponds with this shapely image:
![image](https://github.com/rapidsai/cuspatial/assets/410931/ed8d14cf-9943-4d8a-96c0-25a6f08666e3)

Reply:
Tip: use different characters for the vertices of different geometries. E.g. `x` for the polygon here and `o` for the linestring vertices. And possibly a third character for the "proper" intersections that occur (`*`?).